packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1660,6 +1685,66 @@ class ImportedCluster extends ClusterBase implements ICluster {
     }
     return this.props.clusterEncryptionConfigKeyArn;
   }
+
+  private throwAnErrorIfNoOpenIdConnectIssuerConfigured() {
+    if (!this.clusterOpenIdConnectIssuerUrl || !this.clusterOpenIdConnectIssuer) {
+      throw new Error("You should specify openIdConnectIssuer and " +
+        "openIdConnectIssuerUrl attributes to perform this action");
+    }
+  }
+
+  /**
+   * If OpenID Connect issuer url was configured by clusterAttributes while import,
+   * this will return a value. Otherwise this will be `undefined`
+   * @attribute
+   */
+  public get clusterOpenIdConnectIssuerUrl(): string {
+    return this.attrOpenIdConnectIssuerUrl;
+  }
```

Comment:
This should follow the convention for other attributes in `ImportedCluster`.

```suggestion
  public get clusterOpenIdConnectIssuerUrl(): string {
    if (!props.clusterOpenIdConnectIssuer) {
      throw new Error('"clusterOpenIdConnectIssuer" is not defined for this imported cluster');
    }
    return props.clusterOpenIdConnectIssuer;
  }
```

Same pattern for all the others as well.